VMware.CloudFoundation.PowerManagement.psm1

```diff
@@ -744,6 +744,104 @@ Function Set-VsanClusterPowerStatus {
 }
 Export-ModuleMember -Function Set-VsanClusterPowerStatus
 
+Function Invoke-VxrailClusterShutdown{
+    <#
+        .SYNOPSIS
+        Invoke shut down command on a VxRail Cluster
+
+        .DESCRIPTION
+        The cmdlet will perform a dry run test prior to initiate a shutdown command on a VxRail cluster.
+
+        .EXAMPLE
+        Invoke-VxrailClusterShutdown -server sfo-w01-vxrm.sfo.rainpole.io -user administrator@vsphere.local  -Pass VMw@re1!
+        This example powers off a Vxrail Cluster cluster which the VxRail server sfo-w01-vxrm.sfo.rainpole.io controls. 
+
+    #>
+
+    Param (
+        [Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String]$server,
+        [Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String]$user,
+        [Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String]$pass
+    )
+	
+    Try {
+        Write-PowerManagementLogMessage -Type INFO -Message "Starting the call to the Invoke-VxrailClusterShutdown cmdlet." -Colour Yellow
+
+        $checkServer = (Test-NetConnection -ComputerName $server -Port 443).TcpTestSucceeded
+        if ($checkServer) {
+            Write-PowerManagementLogMessage -Type INFO -Message "Connecting to '$server'..."
+            if ($DefaultVIServers) {
+                Disconnect-VIServer -Server * -Force -Confirm:$false -WarningAction SilentlyContinue  -ErrorAction  SilentlyContinue | Out-Null
```

Comment:
```suggestion
                Disconnect-VIServer -Server * -Force -Confirm:$false -WarningAction SilentlyContinue -ErrorAction  SilentlyContinue | Out-Null
```

Reply:
addressed in commit f8c5a69ef84a1a40675a91879ad0a67701fba108